microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md

```diff
@@ -0,0 +1,175 @@
+
+Additionally, the recorded data can be exported to files on disk. The exported files are called _traces_.
+
+Exporting performance and memory traces is useful when you want to share these files with other people to get help with investigating issues. An exported trace file can be imported in DevTools at any time.
+
+Traces can be exported in one of two ways: 
+
+* As a normal trace: a `.json`, `.heapsnapshot`, `.heapprofile`, or `.heaptimeline` file, which contains minimal information about the runtime data from the traced webpage.
+
+* As an enhanced trace: a `.devtools` file, which contains more runtime data from the traced webpage.
+
+In Microsoft Edge versions prior to 109, enhanced traces aren't available. Starting with 109, the **Performance** and **Memory** tools can export normal and enhanced traces.
+
+
+<!-- ====================================================================== -->
+## Normal traces
+
+A normal trace only contains some of the information that's present in a webpage. Much of the original code is lost and only the recorded performance or memory data of the webpage is preserved in the trace. For example, when importing a `.heapsnapshot` memory trace that was recorded during another DevTools session or on another computer, there's no way to go from an object in the **Memory** tool to its constructor in the **Sources** tool.
+
+Similarly, in the **Performance** tool; when you import a performance trace recorded earlier, it needs to have been recorded on the same site and same configuration to be able to map from the performance report to the JavaScript that was running in the browser.
+
+When a normal trace is imported in DevTools, the recorded data is used to populate the **Performance** or **Memory** tool.
+
+
+<!-- ====================================================================== -->
+## Enhanced traces
+
+An enhanced trace retains more information about the page. For example, an enhanced trace contains the execution context and the list of parsed scripts, and can optionally contain console messages, script sources, and a snapshot of the DOM tree.
+
+When an enhanced trace is imported in DevTools, a new DevTools window appears. This new window isn't connected to the webpage that's running in your browser, and instead re-creates the environment in which the trace was originally recorded.
+
+For example, if a snapshot of the DOM was recorded in the enhanced trace, the **Elements** tool displays this snapshot. If console messages were recorded, the **Console** tool prints these messages. The **Sources** tool displays the scripts that were present during the recording.
+
+Using enhanced traces makes it possible to reliably resolve source code references found in imported traces to the actual runtime code in the **Sources** tool. Additionally, if source maps were presents when a trace was recorded, it will also be possible to resolve code references to their original source code.
```

Comment:
...if source maps were **present** when a trace was recorded, *or if your source maps are [stored on Azure Artifacts Symbol Server](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure)*, it will also...